assets/css/style.css

```diff
@@ -191,36 +185,38 @@ h3 {
 }
 
 .intro h1 {
-  font-size: 25px;
+  width: 98%;
+  font-size: 35px;
   font-weight: 900;
   color: transparent;
+  -webkit-text-stroke: 1px #ec5242;
   background-image: url("../images/orangebg.jpg");
   -webkit-background-clip: text;
 }
```

Comment:
- [ ] I noticed you did change the width, in this case, the font-size is the required change needed, I would encourage you to make it bigger by editing the font-size not the width. Let me know if you dont understand.